README.md

```diff
@@ -185,3 +185,8 @@ Specifically check `ipaddress` and `urllib3` package versions to make sure they
 Starting from 4.0 release, we do not support directly calling exec or attach calls. you should use stream module to call them. so instead
 of `resp = api.connect_get_namespaced_pod_exec(name, ...` you should call `resp = stream(api.connect_get_namespaced_pod_exec, name, ...`.
 See more at [exec example](examples/exec.py).
+
+    Using Stream will overwrite the requests protocol in _core_v1_api.CoreV1Api()_ 
+    This will cause a failure in  non-exec/attach calls. If you reuse your api client object, you will need to 
+    recreate it between api calls that use _stream_ and other api calls.
```

Comment:
the above means `api` needs to get recreated after the exec calls (if there are more non-exec/attach calls)? 
https://github.com/kubernetes-client/python/blob/da8c843bda7ebf03f53cdc95409e5543d2b2abd3/examples/exec.py#L13

Reply:
That is correct.